src/app/search/filter-types.ts

```diff
@@ -77,7 +77,7 @@ export interface FilterDefinition<I, FilterCtx, SuggestionsCtx> {
   destinyVersion?: 1 | 2;
 
   /** methods for retiring a filter */
-  deprecated?: FilterDeprecation;
+  deprecated?: boolean;
```

Comment:
The FilterDeprecation enum wasn't being used for anything, so I replaced it with a simpler boolean for now. We can always go back if we decide to do something with it.

Reply:
fine i guess. changeable when needed.  i would like to see it used more soon, to retire filters and mark them for upconversion